consensus/src/epoch_manager.rs

```diff
@@ -679,7 +681,7 @@ impl EpochManager {
         let initial_data = self
             .storage
             .start()
-            .expect_recovery_data("Consensusdb is corrupted, need to do a backup and restore");
+            .expect_recovery_data("consensusdb is not consistent with aptosdb");
```

Comment:
Is there any response here? i.e., should the user not delete it anymore?

Reply:
they can still delete the db and wait for next epoch, but I want to remove the word "corrupted", "backup & restore" etc